Modelica/Mechanics/Translational.mo

```diff
@@ -1712,6 +1712,125 @@ An eddy current brake reduces the speed of a moving mass. Kinetic energy is conv
 </html>"));
     end EddyCurrentBrake;
 
+    model Vehicle "One-dimensional vehicle with driving resistances"
+      extends Modelica.Icons.Example;
+      import Modelica.Constants.g_n;
+      parameter Modelica.SIunits.Mass m=100 "Mass of vehicle";
+      parameter Modelica.SIunits.Length R=0.25 "Radius of wheel";
+      parameter Modelica.SIunits.Area A=1 "Cross section of vehicle";
+      parameter Real cw=0.5 "Drag resistance coefficient";
```

Comment:
The variable name `cw` is a German thing. In English `cd` may be used, see [Wikipedia](https://en.wikipedia.org/wiki/Drag_coefficient). Possibly a native English speaking person can comment on this. 

@mtiller or @harmanpa Could you possibly comment on this?

Reply:
The commonly understand variable name would be Cd

Reply:
I have also found c_D in https://www.crcpress.com/Road-and-Off-Road-Vehicle-System-Dynamics-Handbook/Mastinu-Ploechl/p/book/9781138075290 (but originating from a German-speaking author anyway ;-) )